rust/eos-rs/src/api/dungeon_mode/dungeon_struct.rs

```diff
@@ -1154,7 +1186,7 @@ impl<'a> GlobalDungeonData<'a> {
 
     /// Get the id of the monster to be randomly spawned.
     pub fn get_monster_id_to_spawn(&mut self, is_for_monster_house: bool) -> MonsterSpeciesId {
-        let weight_id = if is_for_monster_house { 1 } else { 0 };
+        let weight_id = i32::from(is_for_monster_house);
```

Comment:
I think `is_for_monster_house as i32` is a bit more idiomatic?

Reply:
I agree, this was generated by Clippy with --fix, but the as syntax should be better here.